structuralcodes/geometry/_geometry.py

```diff
@@ -199,7 +200,7 @@ def _repr_svg_(self) -> str:
         """Returns the svg representation."""
         return str(self._point._repr_svg_())
 
-    def translate(self, dx: float = 0.0, dy: float = 0.0) -> PointGeometry:
+    def translate(self, dx: float = 0.0, dy: float = 0.0) -> 'PointGeometry':
```

Comment:
Can we use `PointGeometry` instead of `'PointGeometry'` now that you restored future?

Reply:
Fixed in be08c1d.